### ISSUE TITLE
bug: Schedule Scan does not run after creating a new schedule scan task

### ISSUE THREAD

**Issue by S3ntago** [yogeshojha/rengine#1015]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi,

I found that the schedule scan is not working proberl, after creating a new schedule scan, the scan does not run at all, here an example of creating 2 schedule scans and after 24 hrs they still didn't start.
<img width="1139" alt="image" src="https://github.com/yogeshojha/rengine/assets/48615614/b53ee108-0502-4bc7-9508-8c989aae3c6e">

### Expected Behavior

-

### Steps To Reproduce

Create a new schedule scan

### Environment

```markdown
- reNgine: 2.0
- OS: 
- Python: 
- Docker Engine: 
- Docker Compose: 
- Browser:
```

**Comment by NadgobKhan** [comment]
Hello, adding more info regarding this issue. I am running in reNgine v2.0.2.
I noticed whenever the scheduled scan is supposed to start, I get this warning in the logs:
```
rengine-celery-beat-1  | py.warnings | /usr/local/lib/python3.10/dist-packages/django/db/models/fields/__init__.py:1416: RuntimeWarning: DateTimeField PeriodicTask.last_run_at received a naive datetime (2024-02-02 14:58:32.302885) while time zone support is active.
rengine-celery-beat-1  |   warnings.warn("DateTimeField %s received a naive datetime (%s)"
rengine-celery-beat-1  |
```
So it might have something to do with the way the date of the `last_run_at` is sent.

**Comment by yogeshojha** [comment]
Hi @S3ntago thank you for reporting

This will be fixed with #1313 1313
